### PR TITLE
docs(ja): update platform page

### PR DESCRIPTION
**Pull request by morugu**
This PR is a Japanese version of Platform page.

| before | after |
|:---|:---:|
| image | image |

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Comment by toricls** [comment]
Thank you for your contribution @morugu! I'll take a look soon 🤘

File: site/content/docs/include/platform.ja.md [hunk 1]
```diff
@@ -1,4 +1,20 @@
 <div class="separator"></div>
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）
\ No newline at end of file
+<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String or Map</span>  
+`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）  
+例えば`linux/arm64`や`windows/x86_64`などが指定できます。デフォルトは`linux/x86_64`です。
```
Comment (toricls) [inline site/content/docs/include/platform.ja.md]: Adding white-spaces.
```suggestion
例えば `linux/arm64` や `windows/x86_64` などが指定できます。デフォルトは `linux/x86_64` です。
```

File: site/content/docs/include/platform.ja.md [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+
+自動生成された値を上書きして `osfamily` や `architecture` を使うこともできます。
```
Comment (toricls) [inline site/content/docs/include/platform.ja.md]: I think we don't have to fully localize this sentence. Maybe the following is much simpler and good enough for users? :)
```suggestion
異なる `osfamily` や `architecture` を明示的に指定することもできます。
```

File: site/content/docs/include/platform.ja.md [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）
\ No newline at end of file
+<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String or Map</span>  
+`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）  
+例えば`linux/arm64`や`windows/x86_64`などが指定できます。デフォルトは`linux/x86_64`です。
+
+自動生成された値を上書きして `osfamily` や `architecture` を使うこともできます。  
+例えば、Windows ユーザーは以下の設定を
```
Comment (toricls) [inline site/content/docs/include/platform.ja.md]:
```suggestion
例えば、Windows では以下の設定はデフォルトで `WINDOWS_SERVER_2019_CORE` が利用されますが
```

File: site/content/docs/include/platform.ja.md [hunk 4]
```diff
@@ -1,4 +1,20 @@
...
+
+```yaml
+platform: windows/x86_64
+```
+
+Map を使ってデフォルトで `WINDOWS_SERVER_2019_CORE` を利用するように変更できます。
```
Comment (toricls) [inline site/content/docs/include/platform.ja.md]:
```suggestion
Map を使って `WINDOWS_SERVER_2019_FULL` を明示的に指定できます。
```

**Review by toricls — CHANGES_REQUESTED** [review]
Left few comments and suggestions!

File: site/content/docs/include/platform.ja.md (hunk 2 above)
Comment (morugu) [reply site/content/docs/include/platform.ja.md]: I think it's simple and easy to understand! thank you!

**Comment by morugu** [comment]
@toricls Thank you for the review! I committed all your suggestions.

**Review by toricls — APPROVED** [review]

**Comment by toricls** [comment]
Thank you @morugu!

It's your turn @Lou1415926 :)

**Review by Lou1415926 — APPROVED** [review]
Yay thank you @morugu for the contribution and thank you @toricls for reviewing 🎊 !!

**Comment by shingos** [comment]
This PR updates the Japanese localization documentation based on the English documentation after changes to v1.15.0 (62c9781) latest mainline.

By submitting this pull request, I confirm that I can use, modify, copy, and redistribute this contribution under the terms you choose.

**Pull request merged**